source/server/server.h

```diff
@@ -107,25 +107,22 @@ class InstanceUtil : Logger::Loggable<Logger::Id::main> {
                                               Options& options);
 };
 
+class InstanceImpl;
```

Comment:
needed?

Reply:
I thought I needed a forward declaration here but it compiles fine without so will remove it.